radio/util/crossfire-parse.py

```diff
@@ -70,28 +71,47 @@ def dump(data, maxLen=None):
     0xAD, 0x78, 0xD2, 0x07, 0x53, 0x86, 0x2C, 0xF9
 ]
 
+# logs data are BigEndian
+def getCrossfireValue(signed, data=[], *args):
```

Comment:
I have to find a nicer way to do this tomorrow :)

Reply:
Well, you could easily remove the signed argument and assume that 4 bytes data are signed and less then 4 is unsigned, which seems to be true in current version, but as this is more like a tool for debuging, I wanted it to be crystal clear (unless thats not the 'signed' bothering you ?)